Probesys/bileto
# misc: Various patches

Changes proposed in this pull request: see commits messages

How to test the feature manually:

1. Open the user popup menu, go to the "preferences"
2. Click on the "back" button of your browser
3. Check the user menu still opens

 

1. Create an organization
2. Check that you're redirected to the organization page

 

1. Go to the "new organization" and "new ticket" form
2. Check the focus is given to the first input

 

1. Check that icons are still displayed


Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
